.ci/config/matrices.yaml

```diff
@@ -16,6 +16,7 @@ engine-webapp-unit:
   - 'mariadb_106'
   - 'mysql_80'
   - 'oracle_19'
+  - 'oracle_23'
```

Comment:
@tasso94, such change needs to be applied to the EE repo as well:
https://github.com/camunda/camunda-bpm-platform-ee/blob/master/.ci/config/matrices.yaml#L18

Reply:
Done!